Sources/StreamChat/Utils/StreamTimer/StreamTimer.swift

```diff
@@ -0,0 +1,12 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public protocol StreamTimer {
```

Comment:
Oh no... I didn't know we had this already 🤦‍♂️ For now, let's have this duplicate, because the `Timer/RepeatingTimerControl` has static API as well, which is untestable. The problem now is that we have `StreamTimer` and `Timer` both in LLC 😅 

I'll create a tech-debt ticket to fix this issue 👍 
Before we fix this tech debt, @hugobernalstream Can we make both `StreamTimer` and `ScheduledStreamTimer` internal?